week-2/interpret/time-format.js

```diff
@@ -12,32 +12,32 @@ function formatTimeDisplay(seconds) {
   const totalHours = (totalMinutes - remainingMinutes) / 60;
   const remainingHours = totalHours % 24;
 
-  return `${pad(remainingHours)}:${pad(remainingMinutes)}:${pad(
-    remainingSeconds
-  )}`;
+  return `${remainingHours.toString().padStart(2, "0")}:${remainingMinutes
+    .toString()
+    .padStart(2, "0")}:${remainingSeconds.toString().padStart(2, "0")}`;
 }
 
-console.log(formatTimeDisplay(143));
+console.log(formatTimeDisplay(323485723465));
 
 // You can play computer with this example
 // Use the Python Visualiser https://pythontutor.com/visualize.html#mode=edit
 // to help you answer these questions
 
 // Questions
 
-// a) When formatTimeDisplay is called how many times will pad be called?
+// a) When formatTimeDisplay is called how many times will pad be called?                      3 times
 
 // Call formatTimeDisplay with an input of 143, now answer the following:
 
-// b) What value is assigned to the parameter num when pad is called for the first time?
+// b) What value is assigned to the parameter num when pad is called for the first time?       143 assigned to parameter
```

Comment:
Yes, the number 143, in the format of seconds, has been passed to the function. But, when the function pad is called for the first time the value has the same value as the variable remainingHours. Can you predict the value of the variable remainingHours?